content/tags/iteration_tags.md

```diff
@@ -0,0 +1,390 @@
+# Iteration Tags
+
+Iteration Tags are used to run a block of code repeatedly.
+
+1. TOC
+{:toc}
+
+## for
+
+Repeatedly executes a block of code. For a full list of attributes available within a `for` loop, see [forloop(object)](/objects/for_loops/).
+
+`for` loops can output a maximum of 50 results per page. In cases where there are more than 50 results, use the [paginate](/tags/theme_tags/#paginate) tag to split them across multiple pages.
+
+#### Input
+
+~~~ liquid
+  {% for product in collection.products %}
+    {{ product.title }}
+  {% endfor %}
+~~~
+
+#### Output
+
+~~~
+  hat shirt pants
+~~~
+
+## break
+
+Causes the loop to stop iterating when it encounters the `break` tag.
+
+#### Input
+
+~~~ liquid
+  {% for i in (1..5) %}
+    {% if i == 4 %}
+      {% break %}
+    {% else %}
+      {{ i }}
+    {% endif %}
+  {% endfor %}
+~~~
+
+#### Output
+
+~~~
+  1 2 3
+~~~
+
+## continue
+
+Causes the loop to skip the current iteration when it encounters the `continue` tag.
+
+#### Input
+
+~~~ liquid
+  {% for i in (1..5) %}
+    {% if i == 4 %}
+      {% continue %}
+    {% else %}
+      {{ i }}
+    {% endif %}
+  {% endfor %}
+~~~
+
+#### Output
+
+~~~
+1 2 3   5
+~~~
+
+<br>
+
+> ### parameters `for`
```

Comment:
parameters section should be moved right under its parent section (before break and continue)